### ISSUE TITLE
tab-completion broke

### ISSUE THREAD

**Issue by avimar** [Unitech/pm2#2381]
If reporting a bug, please use the following template.

#### Expected behaviour
`pm2 pull be` + `TAB` = `pm2 pull bestfone-api`

#### Actual behaviour
The shell freezes.

When I press ctrl-c, I SEE ` pm2 pull bestfone-api` but when I hit enter, it says: `-bash: m2: command not found`

#### Steps to reproduce
I killed the tab completion from 0.15.10 and re-installed it and reloaded my .bashrc

#### Software versions used

```
OS         : Debian GNU/Linux 8.2 (jessie) 
node.js    : v4.4.4
PM2        : 2.0.12
```

**Comment by pc-jedi** [comment]
Same issue here.

**Software versions used**
```
OS         : Debian GNU/Linux 8.2 (jessie) 
node.js    : v6.6.0
PM2        : 2.0.13
```

**Comment by Unitech** [comment]
thanks for reporting.
@alavit-d could you please double check the fix https://github.com/Unitech/pm2/commit/50c114c81851194785ae7e249cfef4a42117ef14 thanks

**Comment by avimar** [comment]
Worked:

` npm install -g git+https://github.com/Unitech/pm2.git#50c114c81851194785ae7e249cfef4a42117ef14`

**Comment by alavit-d** [comment]
@Unitech all good - added small check in PR https://github.com/Unitech/pm2/pull/2390 for the `pm2 completion` install/uninstall commands who were hanging

**Comment by Unitech** [comment]
alright, available right now in pm2@next, closing

**Comment by Unitech** [comment]
landed in latest public release (2.0.15)

```bash
$ npm install pm2 -g
```
